src/index.js

```diff
@@ -13,13 +13,20 @@ export function screensEnabled() {
 
 export class NativeScreen extends React.Component {
   render() {
-    const { active, style, enabled = true, ...rest } = this.props;
-
+    let { active, activityState, style, enabled = true, ...rest } = this.props;
+    if (active !== undefined && activityState === undefined) {
+      console.warn(
+        'It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens'
```

Comment:
I'm not sure about this warning. This doesn't necessarily mean that it's used by React Navigation. This warning will also appear in 4.x versions of React Navigation libs which won't be easy to upgrade.

Reply:
I added the same warning in `index.native.js`. I think we would like to inform users that upgrading `react-native-screens` without the update of `react-navigation` can result in some bugs. How would you see it?

Reply:
@WoLewicki I thought use of old API should stay the same, no? btw we'll drop use of old API in v6 entirely.

Reply:
On the native side, the old API was removed, the new one can be not entirely compatible.

Reply:
Hmm, maybe consider bumping the version to major if the change isn't backward compatible.